src/scaffold/zome.rs

```diff
@@ -340,7 +340,7 @@ pub fn add_common_zome_dependencies_to_workspace_cargo(
         &format!("={}", hdk_version()),
     )?;
     let file_tree =
-        add_workspace_external_dependency(file_tree, &"serde".to_string(), &"=1.0.166".to_string())?;
+        add_workspace_external_dependency(file_tree, &"serde".to_string(), &"1.0.195".to_string())?;
```

Comment:
This could just be `"1.0"`, it should be fine to let this upgrade now that we're pinning `rmp-serde` which was the reason we ended up pinning here in the first place